src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -49,8 +49,9 @@
               to add to this application.</span>
 
             <BusinessLookup
+              :key="Math.random()"
```

Comment:
@JazzarKarim This resets the instance of this component each time it's used, which resets its state, which allows a used to search anew if there was a problem with the previous search.

Reply:
Karim, this is still not perfect, as it seems to refresh too often, but I don't want to spend any more time on it atm (future fix if needed).

Reply:
Sounds good to me.